scripts/docker/spider.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+# A script to allow for simple spider usage in docker.
+
+DOCKER_WORK_DIR=/usr/src/searchgov-spider
+
+_show_help() {
+    echo "crawl - A simple interface allowing for ad hoc spider runs."
+    echo ""
+    echo "Usage: crawl <domain> <starting-url> [OPTIONS]"
+    echo ""
+    echo "Required Arguments:"
+    echo "domain:           Domain used to limit crawls"
+    echo "starting-url:     URL to start crawling at"
+    echo ""
+    echo "Options:"
+    echo "  --help           Display this help message"
+    echo ""
+    echo "Examples:"
+    echo "   Crawl starting at https://www.gsa.gov, limited to links that are in the www.gsa.gov domain"
+    echo "     - crawl www.gsa.gov https://www.gsa.gov"
+}
+
+_has_args() {
+    if [ "$#" -lt 2 ]; then
+        echo "ERROR: Not Enough Arguments!"
+        echo ""
+        _show_help
+        return 1
+    fi
+}
+
+crawl() {
+    (_has_args $@) || exit 1
+
+    ALLOWED_DOMAINS=$1
+    STARTING_URLS=$2
+
+    cd $DOCKER_WORK_DIR/search_gov_crawler
+    python benchmark.py -d $ALLOWED_DOMAINS -u $STARTING_URLS
```

Comment:
I think after grabbing $1 and $2, the script never forwards the remaining arguments. Anything beyond the starting url is silently dropped, so rest of flags (like --max-depth 3) will never be applied. We should keep the extra args and pass them through, and quote the expansions to avoid word-splitting/globbing
```suggestion
    (_has_args "$@") || exit 1

    ALLOWED_DOMAINS=$1
    STARTING_URLS=$2
    shift 2

    cd "$DOCKER_WORK_DIR/search_gov_crawler" || exit 1
    python benchmark.py -d "$ALLOWED_DOMAINS" -u "$STARTING_URLS" "$@"
```